app/src/main/assets/chart.html

```diff
@@ -15,12 +11,13 @@
     <tr>
       {% for tile in tileRow %}
```

Comment:
Well, since I'm here to comment, I'm really not a fan of this template engine. We should have natural html files (that can be read by the browser even without the data) with clearer code. Html partials to help organization. Complete separation of the view from the logic. I think this is very messy and today we have better ways to generate dynamic html. What are your thoughts on using javascript for this? Don't get me wrong. The way it is is perfectly fine. It works. I just think there are better ways of organizing and generating html.
Java classes could just generate JSON and let webview html + javascript do the rest.